src/gui/contacts/details.sh

```diff
@@ -0,0 +1,36 @@
+gui_contacts_details(){
+    [[ -z "$@" ]] \
+    && message error "<tt>Please select a contact first.<tt>" \
+    || cert_status="Uncertify"; yad --text="<big><tt> \
+                                            $(call cmd_contact_list "$1" \
+                                            | pango_raw \
+                                            | sed 's/[^ ]*/\<b\>&\<\/b\>/') \
+                                            </tt></big>" \
```

Comment:
Use a variable for the value of `--text`.

Reply:
I would prefer to use something better than `cmd_contact_list` for presenting the contact details. Here is one of the places where GTL+Python could be useful. But maybe it can be done with `yad` as well.